sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -482,5 +524,32 @@ public virtual Response ActivateMultiSlot(string eventId, CancellationToken canc
                 throw;
             }
         }
+
+        /// <summary> Gets the configuration details for the live model to use </summary>
+        internal Configuration GetConfigurationForLiveModel(string authType, string authValue)
+        {
+            _personalizerServiceProperties = ServiceConfigurationRestClient.Get();
+            _personalizerPolicy = PolicyRestClient.Get();
+            Configuration config = new Configuration();
+            // set up the model
+            if (authType == "apiKey")
+            {
+                config["http.api.key"] = authValue;
+            }
+            else
+            {
+                //ToDo: Working on changes to support token authentication in RLClient
+                //config["http.token.key"] = authValue;
+            }
+            config["interaction.http.api.host"] = stringEndpoint+"personalizer/v1.1-preview.2/logs/interactions";
+            config["observation.http.api.host"] = stringEndpoint+"personalizer/v1.1-preview.2/logs/observations";
+            config["model.blob.uri"] = stringEndpoint + "personalizer/v1.1-preview.1/model";
```

Comment:
Can you add TODO here to tell we need to make change Rl.Net to support this new model.blob.uri value? It would be good if you also include a task id in the comment.